### ISSUE TITLE
compileAotJava Failed for Actuator and Webflux

### ISSUE THREAD

**Issue by crimsonvspurple** [spring-projects/spring-boot#33278]
A blank project created from the initializr using
- Spring Boot 3 (RC2)
- Actuator
- Webflux

Generated the following error during build. I just started trying out Boot 3. This used to work on Boot 2.

```
> Task :compileAotJava FAILED
<removed>\build\generated\aotSources\org\springframework\boot\actuate\autoconfigure\endpoint\web\reactive\WebFluxEndpointManagementContextConfiguration__BeanDefinitions.java:72: error: ServerCodecConfigurerEndpointObjectMapperBeanPostProcessor has private access in WebFluxEndpointManagementContextConfiguration
  private static BeanInstanceSupplier<WebFluxEndpointManagementContextConfiguration.ServerCodecConfigurerEndpointObjectMapperBeanPostProcessor> getServerCodecConfigurerEndpointObjectMapperBeanPostProcessorInstanceSupplier(
                                                                                   ^
<removed>\build\generated\aotSources\org\springframework\boot\actuate\autoconfigure\endpoint\web\reactive\WebFluxEndpointManagementContextConfiguration__BeanDefinitions.java:74: error: ServerCodecConfigurerEndpointObjectMapperBeanPostProcessor has private access in WebFluxEndpointManagementContextConfiguration
    return BeanInstanceSupplier.<WebFluxEndpointManagementContextConfiguration.ServerCodecConfigurerEndpointObjectMapperBeanPostProcessor>forFactoryMethod(WebFluxEndpointManagementContextConfiguration.class, "serverCodecConfigurerEndpointObjectMapperBeanPostProcessor", EndpointObjectMapper.class);
                                                                              ^
<removed>\build\generated\aotSources\org\springframework\boot\actuate\autoconfigure\endpoint\web\reactive\WebFluxEndpointManagementContextConfiguration__BeanDefinitions.java:82: error: ServerCodecConfigurerEndpointObjectMapperBeanPostProcessor has private access in WebFluxEndpointManagementContextConfiguration
    Class<?> beanType = WebFluxEndpointManagementContextConfiguration.ServerCodecConfigurerEndpointObjectMapperBeanPostProcessor.class;
```

**Comment by crimsonvspurple** [comment]
Just tested with `3.0.0-SNAPSHOT` and it works there. Close if needed.

**Comment by bclozel** [comment]
This has been resolved in #33106
Thanks for trying our RC versions!
